### DISCUSSION TITLE
how to join break line?

### DISCUSSION THREAD

**Discussion by WingDust** [nushell/nushell#10885] [Q&A] [answered]
```sh
echo ['hello', 'world'] | str join '\n'
# hello\nworld
echo ['hello', 'world'] | str join (char -u  '000A')
# this is ugly
```

**Comment by panicbit** [comment, marked as answer]
Single quoted strings don't interpret backslash escapes like `\n`. You need to use double quotes for that:

```nu
['hello', 'world'] | str join "\n"
```
Regarding the `char` command: there are a few aliases for common characters, e.g. for a newline you can use `char nl`. See `char --list` for the full list.
